src/cdk/stepper/stepper.ts

```diff
@@ -72,6 +104,10 @@ export class StepperSelectionEvent {
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class CdkStep implements OnChanges {
+  private _stepperOptions: StepperOptions;
+  _showError: boolean;
```

Comment:
can these be `private`?

Reply:
No because we need to access it in the `CdkStepper` class.